examples/angular/src/pages/ui/primitive-poc/icon/icon.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'amplify-icon-demo',
+  templateUrl: 'icon.component.html',
+})
+export class AmplifyIconExampleComponent {
+  viewstate: { width: string; height: string } = {
```

Comment:
I don't think we need to strictly type this because it's already typed from the icon primitive itself.

```suggestion
  viewstate = {
```

Reply:
[28467b5](https://github.com/aws-amplify/amplify-ui/pull/2436/commits/28467b5846cc0bb40ac862d24bcca96b0e532b1a)